<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComparisonDialog</class>
 <widget class="QDialog" name="ComparisonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ComparisonDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter_compWhole">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget_compOld" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_compOld">
         <property name="text">
          <string>Old Version:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="UpdatedTreeWidget" name="treeWidget_compOld">
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_compNew" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_compNew">
         <property name="text">
          <string>New Version:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="UpdatedTreeWidget" name="treeWidget_compNew">
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_compPart">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget_compOldPart" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_compOldPart">
         <property name="text">
          <string>Old Version:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="UpdatedTreeWidget" name="treeWidget_compOldPart">
         <column>
          <property name="text">
           <string notr="true"/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_compNewPart" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_compNewPart">
         <property name="text">
          <string>New Version:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="UpdatedTreeWidget" name="treeWidget_compNewPart">
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar_comp">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_compOld">
       <property name="text">
        <string>Old File</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_compNew">
       <property name="text">
        <string>New File</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_compSync">
       <property name="text">
        <string>Sync</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_compDiff">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Only Diff</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_compEdit">
       <property name="text">
        <string>Editable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_compComp">
       <property name="text">
        <string>Compare</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_compUpdate">
       <property name="text">
        <string>Update</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_compSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>UpdatedTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>updatedtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
